components/toast/toast.css

```diff
@@ -0,0 +1,34 @@
+.toast {
+  margin: 0.8rem;
+  padding: 1rem;
+  position: fixed;
+  bottom: 0;
+  border-radius: 5px;
+  background-color: var(--grey);
+  display: none;
+  justify-content: center;
+  align-items: center;
```

Comment:
I think justify-content is only applicable for flex containers. You might want to check this once.

Reply:
yes, it works with flex. I am using js to toggle display to flex.

Reply:
Ah got it.